packages/discord.js/src/structures/CommandInteractionOptionResolver.js

```diff
@@ -216,6 +216,17 @@ class CommandInteractionOptionResolver {
     return option?.role ?? null;
   }
 
+  /**
+   * Gets an attachment option.
+   * @param {string} name The name of the option.
+   * @param {boolean} [required=false] Whether to throw an error if the option is not found.
+   * @returns {?MessageAttachment} The value of the option, or null if not set and not required.
+   */
+  getAttachment(name, required = false) {
+    const option = this._getTypedOption(name, 'ATTACHMENT', ['attachment'], required);
```

Comment:
Although there are no merge conflicts, you should probably pull in new changes again, since #7290 changed how this method works:
```suggestion
    const option = this._getTypedOption(name, ApplicationCommandOptionType.Attachment, ['attachment'], required);
```

Reply:
fixed

Reply:
yes, but actually, no. the typings need to be updated too (and you didn't actually pull in the new changes)